test/e2e/user-actions-benchmark.js

```diff
@@ -0,0 +1,143 @@
+const path = require('path');
+const { promises: fs } = require('fs');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const { exitWithError } = require('../../development/lib/exit-with-error');
+const {
+  isWritable,
+  getFirstParentDirectoryThatExists,
+} = require('../helpers/file');
+const { convertToHexValue, withFixtures } = require('./helpers');
+
+const ganacheOptions = {
+  accounts: [
+    {
+      secretKey:
+        '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+      balance: convertToHexValue(25000000000000000000),
+    },
+  ],
+};
+
+async function loadAccount() {
+  let loadingTimes;
+
+  await withFixtures(
+    {
+      fixtures: 'imported-account',
+      ganacheOptions,
+    },
+    async ({ driver }) => {
+      await driver.navigate();
+      await driver.fill('#password', 'correct horse battery staple');
+      await driver.press('#password', driver.Key.ENTER);
+
+      await driver.clickElement('.account-menu__icon');
+      const timestampBeforeAction = new Date();
+      await driver.clickElement({ text: 'Create Account', tag: 'div' });
+      await driver.fill('.new-account-create-form input', '2nd account');
+      await driver.clickElement({ text: 'Create', tag: 'button' });
+      await driver.waitForSelector({
+        css: '.currency-display-component__text',
+        text: '0',
+      });
+      const timestampAfterAction = new Date();
+      loadingTimes = timestampAfterAction - timestampBeforeAction;
+      console.log('end of test');
```

Comment:
Is above `console.log` intended ?

Reply:
oh nice catch!! removing it right now @jpuri thank you!